pkg/cri/resource-manager/policy/policy.go

```diff
@@ -235,6 +239,11 @@ func (p *policy) UpdateResources(c cache.Container) error {
 	return p.backend.UpdateResources(c)
 }
 
+// Rebalance tries to find a more optimal allocation of resources for the current containers.
+func (p *policy) Rebalance() (bool, error) {
+	return p.backend.Rebalance()
```

Comment:
Would `Rebalance()` be relevant/useful for all policies? It'd also be possible to leave it optional and check for 

```
if pr, ok := p.backend.(Rebalance()); ok {
    return pr.Rebalance()
} else {
   return false, nil
}
```

Reply:
Well, we could split out Rebalance() to a policy.Rebalance interface which would not be part of policy.Backend. Then we could do an attempted assertion like that and not call the function if the policy does not support it. So far we have tried to keep the backends identical, without any optional interfaces, and let them provide dummy functions for functionality they prefer not to implement.

Reply:
OK, makes sense, +1